api/jobset/v1alpha2/jobset_types.go

```diff
@@ -134,6 +136,10 @@ type JobSetStatus struct {
 	// RestartsCountTowardsMax tracks the number of times the JobSet has restarted that counts towards the maximum allowed number of restarts.
 	RestartsCountTowardsMax int32 `json:"restartsCountTowardsMax,omitempty"`
 
+	// Phase of the JobSet.
+	// +kubebuilder:default="Running"
```

Comment:
this should be in the status, not the spec, and we can't default it.

Reply:
Currently it is set in jobs status
```go
// JobSetStatus defines the observed state of JobSet
type JobSetStatus struct {
	// +optional
	// +listType=map
	// +listMapKey=type
	Conditions []metav1.Condition `json:"conditions,omitempty"`

         ....
	// Phase of the JobSet.
	// +kubebuilder:default="Running"
	Phase string `json:"phase,omitempty"`

	
}

```